core/src/com/mygdx/insanerpg/Assassin.java

```diff
@@ -0,0 +1,5 @@
+package com.mygdx.insanerpg;
```

Comment:
Create a package called "Model" and then inside make a package called "Unit" then move all the files in there.